src/classes/BGE_ConfigurationWizard_CTRL.cls

```diff
@@ -233,6 +233,20 @@ public with sharing class BGE_ConfigurationWizard_CTRL {
         return dfrs;
     }
 
+/*******************************************************************************************************
+    * @description Returns the corresponding Lightning field type from an Apex field type
+    */
+    public static String getLightningFieldType(String apexFieldType) {
+        if (apexFieldType == 'currency') {
```

Comment:
Can we use "Schema.DisplayType.Currency" (or whatever the correct system type constant) is here instead of the hard-coded string? And actually probably pass in the DisplayType directly instead of the lowercased string.